tools/modules/software/module_samba_lib.sh

```
@@ -0,0 +1,140 @@
+			if [[ -f "/usr/share/samba/smb.conf" ]]; then
+				cp "/usr/share/samba/smb.conf" "/etc/samba/smb.conf"
+			else
+				echo "Warning: Missing configuration file. Use the <configure> option."
+			fi
+		fi
+
+		echo "Samba installed successfully."
+		;;
+		"${commands[2]}")
+		## added subshell to prevent exiting befor removing is complete.
+		$(srv_disable smbd) || echo "No service, skipping"
```

Shellcheck (0.10.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)